### PR TITLE
Chore: APP-1748 Proposals improvements

### PR DESCRIPTION
**Pull request by baxri**
## Description

New proposals are added on the bottom of the ‘Proposal Cards list’ at the Governance Page (e.g. see image 1 below), instead of on top of the list. and general improvements.

https://aragonassociation.atlassian.net/browse/APP-1748

### CODE REVIEW

File: packages/web-app/src/pages/governance.tsx [hunk 1]
```diff
@@ -45,19 +42,24 @@ const Governance: React.FC = () => {
     filterValue !== 'All' ? filterValue : undefined
   );
 
+  // Show proposals loading when loading flag is true and list is empty
+  // Othervise it means there is a show more was clocked
```
Comment (sepehr2github) [inline packages/web-app/src/pages/governance.tsx]: clicked? spelling issue

**Review by sepehr2github — CHANGES_REQUESTED** [review]
Cool, Thanks George, But I found some points such as:
The loading button should be hidden when navigating through the proposal type filter, also filter doesn't work properly

**Comment by baxri** [comment]
> Cool, Thanks George, But I found some points such as: The loading button should be hidden when navigating through the proposal type filter, also filter doesn't work properly

@sepehr2github  good catch, this is problem in current version also of the app. I will fix it.

**Review by sepehr2github — APPROVED** [review]
Thanks, Goerge. Much better now. LGTM 👍

**Pull request merged**